docs/HowTo/Send-Transactions/Concurrent-Private-Transactions.md

```diff
@@ -4,26 +4,35 @@ description: Creating and sending concurrent private transactions with Hyperledg
 
 # Sending concurrent private transactions
 
-Private transaction processing involves two transactions, the private transaction and the [privacy marker transaction].
-The private transaction and the [privacy marker transaction] each have their [own nonce].
+Private transaction processing involves two transactions, the private transaction and the
+[privacy marker transaction (PMT)](../../Concepts/Privacy/Private-Transaction-Processing.md).
```

Comment:
Are we consistent with calling it 'privacy marker transaction'? I know that we are using both, 'privacy' and 'private' when we talk about it. Would be nice to be consistent!

Reply:
Doing a global search on the docs, I think we're consistent with documenting it as "privacy marker transaction" (and typically abbreviating to "PMT").